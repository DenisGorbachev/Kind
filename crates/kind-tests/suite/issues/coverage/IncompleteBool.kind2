type Data.Bool {
    true
    false
}

Or (b: Data.Bool) (bo: Data.Bool) : Data.Bool
Or Data.Bool.true  _          = Data.Bool.true
Or Data.Bool.false Data.Bool.false = Data.Bool.false