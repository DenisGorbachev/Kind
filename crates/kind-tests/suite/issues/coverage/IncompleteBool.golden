   WARN  This function does not cover all the possibilities!

      * Missing case : Data.Bool.false Data.Bool.true 

      /--[suite/issues/coverage/IncompleteBool.kind2:6:1]
      |
    5 |    
    6 |    Or (b: Data.Bool) (bo: Data.Bool) : Data.Bool
      |    v-
      |    \Here!
    7 |    Or Data.Bool.true  _          = Data.Bool.true


